xbmc/ServiceManager.cpp

```diff
@@ -138,38 +141,57 @@ bool CServiceManager::StartAudioEngine()
   return m_ActiveAE->Initialize();
 }
 
-bool CServiceManager::Init3()
+bool CServiceManager::InitStageThree()
 {
+  m_peripherals.reset(new PERIPHERALS::CPeripherals(*m_announcementManager));
   m_peripherals->Initialise();
-  m_PVRManager->Init();
```

Comment:
I care more about the app itself and less about the test suite. I created stage 3 for the reason given.